wormhole-connect/package.json

```diff
@@ -53,7 +53,7 @@
     "web-vitals": "^2.1.4"
   },
   "scripts": {
-    "start": "REACT_APP_CONNECT_ENV=testnet vite",
+    "start": "NODE_OPTIONS=--max-http-header-size=100000 REACT_APP_CONNECT_ENV=testnet vite",
```

Comment:
this just lets us have really long URLs in development so we can permalink large configs (I think I'll have to do the same on Netlify)